docs/next/guides/accessibility/keyboard-shortcuts.md

```diff
@@ -0,0 +1,187 @@
+
+Each keyboard shortcut action is registered in a particular context. There are three contexts:
+
+- `grid` - activated when the user is browsing the data grid (initial)
+- `editor` - activated when the user opens a cell editor
+- `menu` - activated when the user opens the cell's context menu
+
+When the user interacts presses a key or a key combination keyboard, only the actions registered for the active context are executed. Only one context is active at a time.
+
+Please keep in mind that you can get already registered contexts
+using `getContext` method and even register new context using `addContext`. To switch to another context use `setActiveContextName` method.
+
+### Removing keyboard shortcuts
+To remove an already registered keyboard shortcut (such as one of the default keyboard shortcuts), you need to search for it in the relevant context and refer to it by:
+
+- Either the key variant
+- Or the namespace
+
+Use the context's method `removeShortcutByVariants` to remove all shortcuts registered for given key variants (note that it is possible that there are multiple actions registered for a single keyboard shortcut):
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.removeShortcutByVariants([['enter']]);
+```
+
+Use the context's method `removeShortcutByNamespace` to remove all shortcuts registered in a certain namespace:
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.removeShortcutByNamespace('NAMESPACE_ID');
+```
+
+### Adding custom keyboard shortcuts
+You can add **multiple shortcuts** for the same keys in specific context. Simply add it in the end of the shortcuts stack
+using below method
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {} });
+```
+
+If your action must run before a certain other action, you can refer to the other action by its namespace:
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {}, position: 'before', relativeToNamespace: 'ANOTHER_NAMESPACE_ID' });
+```
+
+If your action must run only if some specific precondition is met, you can check for the precondition using a function provided to the `runAction` property:
+
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {}, runAction: () => hot.getSelected() !== void 0 });
+```
+
+### Replacing default keyboard shortcuts
+To replace some keyboards shortcut's action by another one action you have to choose proper context, remove already registered shortcut and register another one.
```

Comment:
@wszymanski This is not what I had in mind when I asked for the instruction to replace default keyboard shortcuts. 

I meant, for example, what change should I make to have Redo triggered by `Shift+Cmd+Z` instead of the Handsontable's default (`Cmd+Y`) on Mac, while keeping `Ctrl+Z` on Windows.